modules/http4s-swagger/src-jvm-ce3/Compat.scala

```diff
@@ -29,14 +30,20 @@ private[smithy4s] object Compat {
     private[smithy4s] type EffectCompat[F[_]] = cats.effect.Concurrent[F]
     private[smithy4s] val EffectCompat = cats.effect.Concurrent
 
-    def docs[F[_]](
-        hasId: HasId,
-        path: String = "docs"
-    )(implicit
-        F: Sync[F]
-    ): HttpRoutes[F] = {
-      val docs = Docs[F](hasId, path)
-      docs.routes
+    def docs: PartiallyAppliedDocs = new PartiallyAppliedDocs("docs")
+    def atPath(path: String): PartiallyAppliedDocs =
```

Comment:
This method should disappear plainly and simply in favour of "smart-setters" in the `PartiallyApplied` construct.